gulpfile.js

```diff
@@ -1204,10 +1215,12 @@ function watchAndSync(options, cb) {
 
 // returns a promise;
 function askDeploy() {
+  // Show user what the currently active firebase project is:
+  execSync('firebase use', {stdio:[0,1,2]});
   prompt.start();
   var schema = {
     name: 'shouldDeploy',
-    description: 'Deploy to Firebase? (y/n)',
+    description: `Deploy ${WWW} to firebase? (y/n)`,
```

Comment:
Yes, its TS syntax for a string with interpolation (i.e., the ${WWW} inside the string).